Instruments/config/2017/Skimmer.cfg

```diff
@@ -161,36 +153,45 @@ file: WJetsToLNu_HT-2500ToInf.root
 [VV]
 merged_output: VV.root
 file_xs: 45.99 WWToLNuQQ_NNPDF31.root WWToLNuQQ_NNPDF31_ext1.root
-file_xs: 11.08 WWTo2L2Nu_ext1.root
-file_xs: 47.73 WWTo4Q_NNPDF31.root WWTo4Q_ext1.root
+file_xs: 45.99 WWTo2L2Nu_NNPDF31_PSweights_ext1.root
+file_xs: 47.73 WWTo4Q_NNPDF31.root
+file_xs: 47.73 WWTo4Q_NNPDF31_PSweights_ext1.root
```

Comment:
`WWTo4Q_NNPDF31.root` and `WWTo4Q_NNPDF31_PSweights_ext1.root` are the same process => should be specified in a single file_xs line